timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -618,8 +625,32 @@ export class TimeGraphChart extends TimeGraphChartLayer {
             if (el instanceof TimeGraphStateComponent && !this.mousePanning && !this.mouseZooming) {
                 this.selectState(el.model);
             }
+            
+            var self = this;
+            this._dblClickTimer = debounce(function(){
+                self._mouseClicks = 0;
+                self._recentlyClickedGlobal = null;
+            }, this._dbClickTime);
```

Comment:
It should not create a new instance of the debounce function every time. Move this declaration to the beginning of the add method, outside of the click event handler.